packages/stark-ui/src/modules/session-ui/pages/session-logout/session-logout-page.component.ts

```diff
@@ -0,0 +1,42 @@
+import {Component, HostBinding, Inject, OnInit, ViewEncapsulation} from "@angular/core";
+
+import {
+	STARK_APP_CONFIG,
+	STARK_LOGGING_SERVICE,
+	StarkApplicationConfig,
+	StarkLoggingService
+} from "@nationalbankbelgium/stark-core";
+
+const componentName: string = "stark-session-logout-page";
+
+/**
+ * Session logout page smart component
+ */
+@Component({
+	selector: "stark-session-logout-page",
+	templateUrl: "./session-logout-page.component.html",
+	encapsulation: ViewEncapsulation.None,
+})
+export class StarkSessionLogoutPageComponent implements OnInit {
+	/**
+	 * Adds class="stark-session-logout-page" attribute on the host component
+	 */
+	@HostBinding("class")
```

Comment:
The class attribute needs to be added in the decorator:

```
@Component({
	selector: ...,
	templateUrl: ...,
	encapsulation: ...,
	host: {
		class: componentName
	}
})
```

Reply:
Actually, it works with HostBinding. And we implemented as this everywhere in stark-ui. 
Should we change the way we declare our components? 😕

Reply:
@SuperITMan indeed, HostBinding works too but it has a drawback, explained here: #664 
therefore we should change this to use the `host` decorator. We will fix them all in another PR ;)